Added maphacks for handsoap in the store in z2a4

fhomolka opened the pull request:
Here's the maphacks I promised to try
I've added the maphack include at the end of voxels.def, I hope that's okay

Dzierzan commented on maphacks/z2a4.mhk:
```diff
@@ -0,0 +1,23 @@
+// Handsoap Bottles in the perfume store
+
+// Left
+sprite 2262 angoff 1536
```
I'd add comment to every line with voxel's name. Take a look at Duke Nukem voxel pack.

fhomolka replied:
Added descriptions to each line

Dzierzan commented on voxels.def:
```diff
@@ -328,4 +328,6 @@ voxel "voxels/3657_VALVE2_G.kvx"		{ tile 3657 }
\ No newline at end of file
+voxel "voxels/3660_SOCKET2.kvx"			{ tile 3660 }
+
+include maphacks.def
```
I would add `**include**` on top of everything here.

fhomolka replied:
moved include on top. Wasn't sure if voxels had to be loaded prior

fhomolka committed:
Corrected where maphacks are loaded from in voxels.def

Added line-per-line descriptions for handsoap maphacks, positioned include at the top

added maphacks for handsoap in the store in z2a4

Dzierzan commented:
LGTM. Just squash the commits and I'll merge it.

fhomolka commented:
Okay, I did some stupid things in the process, but they should be squashed now
